moc_sprint_tools/create_sprint_boards.py

```diff
@@ -43,8 +43,7 @@ def copy_card(source_card, destination_column):
             LOG.warning('Couldn\'t copy card with unkown type')
             return
 
-        LOG.info('adding card "%s" to %s', (content.title,
-                                            destination_column.name))
```

Comment:
This was generating an exception because you were passing a single argument (a tuple) instead of two arguments.

Reply:
Ah, missed the `%`.

Reply:
The logging methods actually work differently: it applies the formatting internally, so you don't use `%` in the call itself.